src/pcscdaemon.c

```diff
@@ -623,11 +623,6 @@ int main(int argc, char **argv)
 				"cannot create " PCSCLITE_IPC_DIR ": %s", strerror(errno));
 			return EXIT_FAILURE;
 		}
-
-		/* set mode so that the directory is world readable and
-		 * executable even is umask is restrictive
-		 * The directory containes files used by libpcsclite */
-		(void)chmod(PCSCLITE_IPC_DIR, mode);
 	}
```

Comment:
Why do you think this chmod() call can be removed?

Reply:
Few lines above you are creating new dir and than you call this chmod to change same permissions as were set at the creation of dir.